userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -60,19 +68,22 @@ Downgrade:: The change of a subscription from one plan to another plan of lesser
 
 {A downgrade is a change order or amendment to an existing contract or subscription in which a customer chooses a lower service level. A downgrade can be a less expensive service offering, or a smaller quantity or users or usage levels. Depending on business rules, this change can result in a prorated credit of the original service or a penalty fee associated with the change.}
 
-//Dunning
-//{The process of handling failed payments and declined credit cards to recover lost revenue.}
+Dunning:: See <<overdue_system, Overdue System>>.
 
 Duration:: Defines the length of time for a <<phase,phase>>. Kill Bill supports durations of days, months, years or unlimited.
 
 == E
 
+[[entitlement]]
+Entitlement:: Represents the service provided to an end user (customer). It is separate from billing. See <<subscription, Subscription>>.
```

Comment:
Should we clarify this further? 

"Represents access to the product/service provided to an end user (customer). It is ... "